components/IngredientsList.js

```diff
@@ -80,11 +82,13 @@ const List = styled.ul`
   justify-content: center;
 `;
 
-const ListItem = styled.li`
+const StyledListItem = styled.a`
```

Comment:
You should have an `li` as the direct child of a list.

This `li` could contain a link to the detail pages.

Since it links between pages in your application, you should use the Link component from Next.js